meerk40t/lihuiyu/device.py

```diff
@@ -21,6 +21,7 @@ class LihuiyuDevice(Service, ViewPort):
     """
     LihuiyuDevice is driver for the M2 Nano and other classes of Lihuiyu boards.
     """
+    priority = 1
```

Comment:
device.priority is increased at the class level. This value can be checked and viewed prior to initializing the class.